contents/handbook/people/team-structure/team-structure.md

```diff
@@ -51,3 +53,24 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 ## [People & culture](people)
 
 - [Eltje Lange](/handbook/people/team#eltje-lange-people-and-talent) (People and Talent)
+
+
+# Organization changes
+
+We’re still an early stage company, and so is our product. We strongly value moving and shipping fast (see core values) and we constantly iterate not only our product but our organization too. As this happens, it may be quite possible that organizational changes occur. This mostly happens on the small teams plane. To make the changes smoother for everyone, here’s the checklist we use:
+
+- [ ] Discuss with concerned team leads (and/or managers if applicable).
+- [ ] Discuss with concerned team member(s).
+- [ ] Open a PR on `posthog.com` documenting the change on the handbook.
+    - Update the [Team Structure](/handbook/people/team-structure/team-structure) page.
+    - Update the relevant small teams pages'.
+- [ ] Let everyone else in the company know in the next PostHog News session.
+- [ ] Add/remove from relevant [Sentry teams](https://sentry.io/settings/posthog/teams/)
+- [ ] Add/remove from relevant [GitHub teams](https://github.com/orgs/PostHog/teams)
```

Comment:
I don't think we really use these, but we have them. If we're not going to keep using them, perhaps worth deleting them to avoid any confusion?

Reply:
@timgl